client/securedrop_client/gui/source/delete/dialog.py

```diff
@@ -31,31 +31,44 @@
 class DeleteSourceDialog(ModalDialog):
     """Used to confirm deletion of source accounts."""
 
-    def __init__(self, sources: list[Source]) -> None:
+    def __init__(self, sources: list[Source], source_total: int) -> None:
         super().__init__(show_header=False, dangerous=True)
         self.sources = sources
+        self.source_total = source_total
 
         # If the dialog is constructed with no sources, show a warning; otherwise,
         # confirm the number and designation of the sources to be deleted
         num_sources = len(sources)
         if num_sources == 0:
             self._show_warning_nothing_selected()
         else:
-            continue_text = ngettext(
-                "YES, DELETE ENTIRE SOURCE ACCOUNT",
-                "YES, DELETE {number} SOURCE ACCOUNTS",
-                num_sources,
-            ).format(number=num_sources)
-
-            self.body.setText(self.make_body_text(self.sources))
+            if num_sources < source_total:
+                continue_text = ngettext(
+                    "YES, DELETE ENTIRE SOURCE ACCOUNT",
+                    "YES, DELETE {number} SOURCE ACCOUNTS",
+                    num_sources,
+                ).format(number=num_sources)
+            else:
+                continue_text = ngettext(
+                    "YES, DELETE ENTIRE SOURCE ACCOUNT",
+                    "YES, DELETE ALL {number} SOURCE ACCOUNTS",
+                    num_sources,
+                ).format(number=num_sources)
+
+            self.body.setText(self.make_body_text(self.sources, self.source_total))
             self.continue_button.setText(continue_text)
             self.cancel_button.setDefault(True)
             self.cancel_button.setFocus()
             self.confirmation_label.setText(_("Are you sure this is what you want?"))
             self.adjustSize()
 
-    def make_body_text(self, sources: list[Source]) -> str:
-        message_tuple = (
+    def make_body_text(self, sources: list[Source], source_total) -> str:
+        if len(sources) == source_total:
+            all_sources_text = ("<p><b>", _("Notice: All sources have been selected!"), "</p></b>")
```

Comment:
"Notice" is a little clunky here, somewhere between a "Posted: No Trespassing" sign and a command.  :-)  How about?

```suggestion
            all_sources_text = ("<p><b>", _("WARNING: All sources have been selected!"), "</p></b>")
```

Or just:

```suggestion
            all_sources_text = ("<p><b>", _("All sources have been selected!"), "</p></b>")
```

Reply:
I tried `Warning:` first but it felt a little overwrought - there are legitimate cases where you might wanna nuke the dataset, hence `Notice:`.